index.html

```diff
@@ -41,6 +41,21 @@ <h1>Introducing Karma</h1>
         <p>Bring WiFi with you, everywhere you go</p>
         <button>Learn More</button>
       </article>
+      <h1>Everyone needs a little Karma.</h1>
+      <section class="device">
```

Comment:
You don't seem to refer to the classes "device", "boost" or "pay" in your css file.

Reply:
I've changed all section class to card and use it in .card CSS to style.